src/background/controllers/accountController.ts

```diff
@@ -124,10 +128,12 @@ export default class AccountController extends IController {
   * @param mnemonic The mnemonic to derive the wallet from.
   */
   public importMnemonic = async (accountName: string, mnemonic: string) => {
-    if (!mnemonic) {
-      throw Error('invalid mnemonic'); }
-
     try {
+      if (!mnemonic) {
+        // This is already validated on the popup ui, and should never hit
+        throw Error('invalid mnemonic');
```

Comment:
nit: use an assertion. 
https://www.w3schools.com/nodejs/ref_assert.asp
https://nodejs.org/api/assert.html#assert_assert_value_message

assertions in general are meant for code that should not be reachable, if it is then it should terminate.